File: pi/pi.js
var num = 4.0;
var pi = 0;
var plus = true;
var startTime = Date.now();
var den = 1;
while (den < 100000000){
	if (plus){
		pi = pi + num/den;
    	plus = false;
	}    
  else{
	pi = pi - num/den;
    plus = true;
  }
  den = den + 2;
}

console.log("PI = " + pi +"\n");              // calculated value of pi
console.log("Math::PI = " + Math.PI + "\n");  // pi from the math class
var endTime = Date.now();
console.log((endTime - startTime) / 1000);

File: pi/pi.go
package main

import "fmt" 
import "time"
import "math"

func main() {
  num := 4.0;
  pi := 0.0;
  plus := true;
  startTime := time.Now().UnixNano();
  den := 1.0;
  for den < 100000000 {
    if plus {
      pi = pi + num/den;
      plus = false;
    } else {
      pi = pi - num/den;
      plus = true;
    }
    den = den + 2;
  }
  fmt.Printf("PI = ");
  fmt.Printf("%v\n", pi) 
  fmt.Printf("Math::PI = ");
  fmt.Printf("%v\n", math.Pi);
  endTime := time.Now().UnixNano();
  fmt.Printf("%v\n", float64(endTime - startTime) / 1000000000);
}
